resources/contribute-blog/_blog-template.md

```diff
@@ -0,0 +1,54 @@
+---
+category: 2025
+date: 2025-mm-dd
+seo:
+    title: SEO optimized title under 50chars
+    description: SEO optimized description for the blog post that can be displayed in search results.
+
+labels:
+    - General
+markdown:
+    editPage:
+        hide: true
+---
+# SEO optimized title under 50chars (Recommend keeping it same as the meta title above for consistency)
+
+
+Congrats on taking the first step toward drafting your blog post! Use this template as a starting point to organize your blog post.
+
+<!-- BREAK -->
+
+Refer to the [Contributing Guide](https://xrpl.org/resources/contribute-documentation) for detailed guidelines and examples.
+
+## Section: Handy Reference
+
+To help you get started quickly, here’s a handy guide to the syntax for some of the most frequently used components in a blog post — perfect for streamlining your writing process.
+
+To include an internal link, use a relative path.  For example, here's a link to the [Contributing Guide](../resources/contribute-documentation/index.md).
```

Comment:
For example, to create a link to the Contribute Documentation topic, you would use the following syntax: